api/services/ocean/spark/parameters/query/oceanSparkClusterForceDelete.yaml

```diff
@@ -0,0 +1,7 @@
+in: query
+name: forceDelete
```

Comment:
@thorsteinnth it looks like this query param is not yet used in `wave-core`. Should we remove it from the doc?

Reply:
this parameter will not be used yet, we shouldn't document it

Reply:
Removed, thanks!